flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/restart/FixedDelayRestartStrategy.java

```diff
@@ -82,7 +82,7 @@ public static FixedDelayRestartStrategyFactory createFactory(Configuration confi
 		int maxAttempts = configuration.getInteger(ConfigConstants.RESTART_STRATEGY_FIXED_DELAY_ATTEMPTS, 1);
 
 		String timeoutString = configuration.getString(
-			AkkaOptions.WATCH_HEARTBEAT_INTERVAL);
+			AkkaOptions.WATCH_HEARTBEAT_PAUSE);
```

Comment:
We cannot make this change, this introduces crazy delay on each recovery.

Reply:
I will fix the inconsistency in other way: fixing exception message instead of this getter.